REQUIRE

```diff
@@ -1,6 +1,5 @@
-julia 0.6
```

Comment:
why delete the julia version completely instead of saying `1.0` ?

Reply:
Originally I had deleted the REQUIRE file altogether...Not sure why we even need a REQUIRE anymore since we're using Project.toml files.  Then travis started choking about the `Dates` package, and when I realized that it was running an older version of Julia that needs the file, I put it back and forgot the 1.0.  Now that you are pointing it out, wondering how this would work in any case with `julia 1.0` when I am testing it on `1.0.3`, `1.1.0` and `nightly`.  

All in all, it seems the only reason to keep it has something to do with METADATA and getting the package registered. See discourse: [Current best practices for depencency management for registered packages](https://discourse.julialang.org/t/current-best-practices-for-depencency-management-for-registered-packages/20376).

Your call, perhaps you better understand this subtle point.

Reply:
maybe just add a `julia 1.0` for good measure